docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

```diff
@@ -0,0 +1,189 @@
+---
+id: camunda-exporter
+title: "Camunda exporter"
+sidebar_label: "Camunda Exporter"
+description: "The common camunda exporter exporting zeebe records to harmonized web-apps indices ."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The Camunda Exporter replaces the old Elasticsearch and Opensearch exporters. Previously, these exporters mainly
+exported raw records from Zeebe streams into several indices. Camunda webapps (e.g., Operate, Tasklist) then imported
+and transformed these records into useful forms. The Camunda Exporter aims to write records directly into
+Elasticsearch/Opensearch in the form needed by the webapps, eliminating the need for additional importing or
+transformation. This increases process instance throughput and reduces the latency of changes appearing
+in Operate and Tasklist webapps by cutting off the importer from the data pipeline.
+
+:::note
+The indexes are created as required, and will not be created twice if they already exist. However, once disabled, they
+will not be deleted (that is up to the administrator.) Similarly, data is never deleted by the exporter, and must be
+deleted by the administrator when it is safe to do so.
+A [retention](#retention) policy can be configured to automatically delete data after a certain number of days.
+:::
+
+## Configuration
+
+:::note
+As the exporter is packaged with Zeebe, it is not necessary to specify a `jarPath`.
+:::
```

Comment:
🙃 I feel like these notes are more confusing than helpful. If I read this guide the first time I don;t know what a jarPath is.

Reply:
Maybe we move it downwards